github-react-app/src/components/RepoTile/RepoTile.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import Avatar from "@components/Avatar";
+import StarIcon from "@components/StarIcon";
+import { RepoItem } from "@store/GitHubStore/types";
+
+type RepoTileProps = {
+    onClick?: (e: React.MouseEvent) => void;
```

Comment:
Лучше передавать вот такой callback + сделать его обязательным(в 4-ой лекции рассмотрим `useCallback`), таким образом избавимся от лишних рендеров
```suggestion
    onClick: (id: string) => void;
```

а внутри компонент вызовете `onClick(item.id)`